plugins/vpc-shared-eni/plugin/commands.go

```diff
@@ -33,6 +36,17 @@ func (plugin *Plugin) Add(args *cniSkel.CmdArgs) error {
 		return err
 	}
 
+	// Parse orchestrator-specific configuration.
```

Comment:
Why do we need to move this block from config to commands.go? We can pass an "IsAdd" to config.New(args) call and have the logic execute only for ADD commands.